posting.go

```diff
@@ -368,15 +372,19 @@ func (i *PostingsIterator) Size() int {
 // the freqNorm and location specific information
 // of a hit
 func (i *PostingsIterator) ResetBytesRead(val uint64) {
-	atomic.StoreUint64(&i.bytesRead, val)
+	if AccountIOStats {
+		atomic.StoreUint64(&i.bytesRead, val)
+	}
 }
 
 func (i *PostingsIterator) BytesRead() uint64 {
```

Comment:
Worth adding the flag here and everywhere else for BytesRead and BytesWritten as well to avoid the atomic op.